production-repos.json

```diff
@@ -1,311 +1,13 @@
-  "rpe-pdf-service",
-  "probate-submit-service",
-  "lau-idam-backend",
-  "disposer-idam-user",
-  "sscs-shared-infrastructure",
-  "et-pet-shared-infrastructure",
-  "rpx-xui-webapp",
-  "cnp-rhubarb-recipes-service",
-  "sptribs-shared-infrastructure",
-  "ccd-definition-store-api",
-  "darts-automation",
-  "pre-functions",
-  "pre-portal",
-  "finrem-shared-infrastructure",
-  "fis-shared-infrastructure",
-  "ccd-case-management-web",
-  "bulk-scan-orchestrator",
-  "ccd-case-document-am-api",
-  "am-role-assignment-batch-service",
-  "ccpay-functions-node",
-  "cnp-module-api-mgmt-subscription",
-  "sscs-task-configuration",
-  "terraform-module-servicebus-topic",
-  "civil-service",
-  "probate-caveats-frontend",
-  "prl-cos-api",
-  "ccd-admin-web",
-  "pcq-loader",
-  "terraform-module-servicebus-subscription",
-  "ccd-logstash",
-  "probate-shared-infrastructure",
-  "juror-shared-infrastructure",
-  "idam-web-public",
-  "hmc-cft-hearing-service",
-  "wa-shared-infrastructure",
-  "cmc-claim-submit-api",
-  "cmc-shared-infrastructure",
-  "rpx-xui-manage-organisations",
-  "rpx-shared-infrastructure",
-  "ia-hearings-api",
-  "hmi-rota-dtu",
-  "cui-ra",
-  "cnp-idam-compute",
-  "civil-ccd-definition",
-  "cpo-case-payment-orders-api",
-  "em-annotation-api",
-  "rd-commondata-dataload",
-  "dg-docassembly-api",
-  "camunda-bpm",
-  "cnp-idam-vault",
-  "cnp-module-storage-account",
-  "pip-shared-infrastructure-bootstap",
-  "rd-location-ref-data-load",
-  "terraform-module-servicebus-queue",
-  "ia-case-documents-api",
-  "rd-caseworker-ref-api",
-  "bar-shared-infrastructure",
-  "reform-scan-notification-service",
-  "ccd-api-gateway",
-  "et-shared-infrastructure",
-  "sds-toffee-recipes-service",
-  "prl-wa-task-configuration",
-  "darts-proxy",
-  "pdda-shared-infrastructure",
-  "cnp-module-storage",
-  "ia-task-configuration",
-  "fpl-wa-task-configuration",
-  "rpe-shared-infrastructure",
-  "makeaplea-shared-infrastructure",
-  "pip-data-management",
-  "em-hrs-api",
-  "ccd-shared-infrastructure",
-  "c100-shared-infrastructure",
-  "darts-gateway",
-  "darts-shared-infrastructure",
-  "am-role-assignment-refresh-batch",
-  "ccd-user-profile-api",
-  "ia-timed-event-service",
-  "cmc-claim-store",
-  "hmi-shared-infrastructures-bootstrap",
-  "probatemandb",
-  "idam-api",
-  "ccfr-fees-register-app",
-  "cnp-module-vnet",
-  "prd-pui-registration",
-  "pdm-shared-infrastructure",
-  "lau-frontend",
-  "pip-subscription-management",
-  "sscs-hearings-api",
-  "fact-admin",
-  "pcq-consolidation-service",
-  "rpa-professional-api",
-  "civil-camunda-bpmn-definition",
-  "rpa-jui-webapp",
-  "cmc-citizen-frontend",
-  "pcq-frontend",
-  "cnp-idam-master",
-  "juror-public",
-  "cnp-module-palo-alto",
-  "help-with-fees-shared-infrastructure",
-  "service-auth-provider-app",
-  "cnp-rhubarb-frontend",
-  "cnp-plum-shared-infrastructure",
-  "wa-workflow-api"
+    "test-repo-1",
+    "test-repo-1ew34nh9",
```

Comment:
you are attempting to pass in test data?

Reply:
Yeah this is test data from the repos we've created in the hmcts-test org. These are just used as a test, we will swaap these out for the prod repo list once ready.

Reply:
why not swap with prod repo as part of PR?